niceml/data/netdataloggers/netdatalogger.py

```diff
@@ -17,6 +17,7 @@ class NetDataLogger(ABC):
     """Abstract implementation of an NetDataLogger"""
 
     def __init__(self):
+        """Constructor"""
```

Comment:
Please add more details. I think it's fine if you use the class docstring in combination with a description of the attributes.